<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoMTimeCheckDialog</class>
 <widget class="QDialog" name="LoMTimeCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check LoM land timer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelGuild">
       <property name="text">
        <string>Guild</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxGuild">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAccount">
       <property name="text">
        <string>Account</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxAccount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxPostToGuildChat">
       <property name="text">
        <string>post message to guild chat?</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonRun">
       <property name="text">
        <string>Run</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>gfx/run.png</normaloff>gfx/run.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEditLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxGuild</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LoMTimeCheckDialog</receiver>
   <slot>onGuildIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRun</sender>
   <signal>clicked()</signal>
   <receiver>LoMTimeCheckDialog</receiver>
   <slot>onRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked()</signal>
   <receiver>LoMTimeCheckDialog</receiver>
   <slot>onStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onGuildIndexChanged(int)</slot>
  <slot>onRun()</slot>
  <slot>onStop()</slot>
 </slots>
</ui>
